examples/build/main.go

```diff
@@ -9,6 +9,7 @@ import (
 	"os"
 	"runtime"
 
+	"braces.dev/errtrace"
```

Comment:
Not sure we need `errtrace` in an example, but I don't object to it either.

Reply:
that was automatically added by their tool, agree it is not necessary for example.